webui/react/src/stores/userSettings.tsx

```diff
@@ -39,11 +37,12 @@ class UserSettingsStore extends PollingStore {
    *          (or has been removed) this is `null`.
    */
   public get<T>(type: t.Type<T>, key: string): Observable<Loadable<T | null>>;
+  public get<T>(type: t.Decoder<Json, T>, key: string): Observable<Loadable<T | null>>;
```

Comment:
Forgot that the last overload line is the implementation and isn't available for calling.